lib/spi/types.ts

```diff
@@ -110,7 +110,8 @@ export enum GlobalResources {
     Vpc = 'vpc',
     HostedZone = 'hosted-zone',
     Certificate = 'certificate',
-    KmsKey = 'kms-key'
+    KmsKey = 'kms-key',
+    S3 = 's3',
```

Comment:
I am not sure we need S3 in global resources that in fact is pointing to single bucket. it is as if we stating "the bucket for the blueprint" which I don't think there is such a concept, as opposed to "the VPC", "the KMS Key", etc. I assume customers will use bespoke names like `"my-bucket", new CreateS3BucketProvider` and such.

Reply:
addressed